docs/runbooks/VirtOperatorRESTErrorsBurst.md

```diff
@@ -2,24 +2,53 @@
 
 ## Meaning
 
-More than 80% of the rest calls in virt-operator pods failed in the last 5 minutes. 
+More than 80% of REST calls in `virt-operator` pods failed in the last 5 minutes. 
+
+The `virt-operator` has likely lost the connection to the API server. 
 
 ## Impact
 
-This alert suggests that the virt-operator is likely lost the connection to the apiserver. Cluster-level actions such as upgrading and controller reconciliation will likely not function. Customer workloads, i.e. VM and VMI, should not be affected.
+Cluster-level actions, such as upgrading and controller reconciliation, probably do not function. 
+
+However, customer workloads, such as virtual machines (VMs) and VMIs, are not likely to be affected.
 
 ## Diagnosis
 
-Two common types of errors may lead to this alert:
-- The apiserver is overloaded and we run into timeouts. Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls. For users without cluster privileges, logs of KubeVirt apiserver pods can be fetched with `kubectl logs` command.
-- The virt-operator pod cannot reach the apiserver. Common issues are network connectivity issues such as DNS issues on the node. Check virt-operator logs to verify whether it can connect to the apiserver at all.
-    - `export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"`
-    - `kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-operator`
-    - `kubectl -n $NAMESPACE logs <pod-name>`.
-    - `kubectl -n $NAMESPACE describe pod <pod-name>`.
+This error is most frequently caused by one of the following problems:
```

Comment:
I would move this bullet list to the "Meaning" section because it is conceptual, not procedural.